minimisers/simplex/fMinSearch.m

```diff
@@ -334,7 +351,8 @@
 %         fprintf('%s \n', 'func_evals = ')
 %         fprintf('%s \n', num2str(func_evals))
     end
-    if rem(itercount, controls.updatePlotFreq) == 0   
+    controls.calcSLD = false;
+    if doPlotEvent && rem(itercount, controls.updatePlotFreq) == 0   
```

Comment:
Again this implies that we only need the final evaluation from the while loop, but I can see that that may be very difficult to determine.

Reply:
Yes, for now I will leave as is except if optimization is needed